test/bundler_smoke_tests/lib/run-webpack.js

```diff
@@ -1,16 +1,11 @@
 import webpack from "webpack";
-import path from "node:path";
-import { fileURLToPath } from "node:url";
-
-const dirname = path.dirname( fileURLToPath( import.meta.url ) );
-const configPath = path.resolve( dirname, "..", "webpack.config.cjs" );
 
 // See https://webpack.js.org/api/node/#webpack
 export async function runWebpack() {
 	return new Promise( async( resolve, reject ) => {
 		console.log( "Running Webpack" );
 
-		const { default: config } = await import( configPath );
+		const { default: config } = await import( "../webpack.config.cjs" );
```

Comment:
A full path with `c:` cannot be passed to import. Using the relative path instead was the least complicated solution.

Reply:
TIL!